extensions/amp-analytics/analytics-vars.md

```diff
@@ -86,6 +86,12 @@ Provides the referrer where the user came from. It is read from `document.referr
 
 Example value: `https://www.google.com`
 
+### sourceUrl
+
+Parses and provides the source URL of the current document to the URL.
+
+The source URL is extracted from the proxy URL if the document is being served from a proxy. Otherwise the original document URL is returned. For instance, if the URL is served via the proxy `https://cdn.ampproject.org` from the URL `https://cdn.ampproject.org/c/s/example.com/page.html`, then `SOURCE_URL` would return `https://example.com/page.html`.
```

Comment:
Could you also document behavior if not being served from a proxy that is https://cdn.ampproject.org?

This will (a) make it even clearer that this case exists so people can take note and (b) fully explain behavior.